db/migrations/daily_tasks.sql

```diff
@@ -0,0 +1,49 @@
+-- phpMyAdmin SQL Dump
```

Comment:
Can you add the date/time to this file name in format `yyyymmdd_hhmm_daily_tasks.sql`? and add the line

`update system_storage set database_version='yyyymmdd_hhmm';`
 Going to make some migration tooling that relies on prefixed file names.
 
 Also, add a row to add the `boost_time` column to teams table - Aiming for migrations to be painless/largely automated for everyone working on this